test/apps/iodemo/ucx_wrapper.cc

```diff
@@ -529,11 +529,17 @@ UcxContext::wait_completion(ucs_status_ptr_t status_ptr, const char *title,
 
 void UcxContext::recv_io_message()
 {
-    ucs_status_ptr_t status_ptr = ucp_tag_recv_nb(_worker, &_iomsg_buffer[0],
-                                                  _iomsg_buffer.size(),
-                                                  ucp_dt_make_contig(1),
-                                                  IOMSG_TAG, IOMSG_TAG,
-                                                  iomsg_recv_callback);
+    ucp_request_param_t param;
```

Comment:
can we make param a global var (static class member) so it will not consume stack space?

Reply:
Sounds good. I'll do it at next weekend.